.github/workflows/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.18
+
+ARG version
+ARG fork
+ENV version=${version}
+ENV fork=${fork:-scylladb/scylla-bench}
+
+RUN apt-get update; apt-get -y install unzip curl
+
+RUN curl -Lo sb.zip https://github.com/${fork}/archive/refs/${version}.zip ;\
```

Comment:
You really should:
1. Use && - there's no point in continuing commands if the former failed
2. Cleanup the zip file at the end.

(we'd like to keep this container small)